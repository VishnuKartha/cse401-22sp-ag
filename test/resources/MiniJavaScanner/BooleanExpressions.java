boolean a = true;
boolean b = !a;