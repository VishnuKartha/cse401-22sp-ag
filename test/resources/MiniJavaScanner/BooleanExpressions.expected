BOOLEAN ID(a) BECOMES TRUE SEMICOLON BOOLEAN ID(b) BECOMES NOT ID(a) SEMICOLON